test/test_stieltjes.jl

```diff
@@ -141,6 +141,15 @@ end
             @test H * ChebyshevTWeight() ≡ QuasiZeros{Float64}((x,))
             @test H * ChebyshevUWeight() == π*x
             @test (H * LegendreWeight())[0.1] ≈ log((0.1+1)/(1-0.1))
+            
+            x = axes(Laguerre(), 1)
+            @test (inv.(x .- x') * LaguerreWeight())[0.1] ≈ exp(-0.1)*expinti(0.1)
+            @test isapprox((inv.(x .- x') * LaguerreWeight(-0.5))[0.1],  3.317769, rtol=1e-4) # Mathematica
```

Comment:
Why are you only using 4 digits from Mathematica? Here's how you can get 19 digits:

<img width="649" alt="image" src="https://user-images.githubusercontent.com/1229737/203831450-a258c3ec-9dbb-482c-8049-93fc38ea5573.png">

Reply:
Ah, this works fine! Whatever I was doing before with NIntegrate was struggling to converge.